modules/calib/include/opencv2/calib.hpp

```diff
@@ -1099,6 +1099,34 @@ CV_EXPORTS_W double stereoCalibrate( InputArrayOfArrays objectPoints,
                                      int flags = CALIB_FIX_INTRINSIC,
                                      TermCriteria criteria = TermCriteria(TermCriteria::COUNT+TermCriteria::EPS, 100, 1e-6) );
 
+/*
+Multiview calibraton.
+
+Input:
+@objPoints   of size NUM_FRAMES x NUM_POINTS x 3 -- object points
+@imagePoints of size NUM_CAMERAS x NUM_FRAMES x NUM_POINTS x 3 -- image points
+@imageSize   of size NUM_CAMERAS x 2 -- size of images
+@visibility - visibility mask of size NUM_CAMERAS x NUM_FRAMES - boolean
+@USE_INTRINSICS_GUESS - if true then intrinsic parameters (K and distortion) are not estimated
+@is_fisheye of size NUM_CAMERAS -- indicates whether i-th camera is fisheye. Currently supported either all or none.
+
+Output:
+@return: true if success, false - otherwise
+@Rs of size NUM_CAMERAS x 3 x 3 -- rotation matrices wrt camera 0, where Rs[0] = I
```

Comment:
Please document axis direction.

Reply:
Rs will be array of rotation vectors: https://github.com/opencv/opencv/pull/22363/commits/822cfa465d73035bad85e33c7b2e5821e65779be